charts/cluster-api-runtime-extensions-nutanix/templates/cni/cilium/manifests/helm-addon-installation.yaml

```
@@ -12,6 +12,7 @@ data:
       chainingMode: portmap
       exclusive: false
     hubble:
+      relay: enabled
```

Ran locally and cilium install failed with (taken from Cilium HCP):

```
values don't meet the specifications of the schema(s) in the following chart(s):
cilium:
- hubble.relay: Invalid type. Expected: object, given: string
```
```suggestion
      relay:
        enabled: true
```